src/Widget/ErrorSummary.php

```diff
@@ -178,8 +193,10 @@ private function collectErrors(): array
         $errors = HtmlFormErrors::getErrorSummaryFirstErrors($this->formModel);
         $errorMessages = [];
 
-        if ($this->showAllErrors) {
+        if ($this->showAllErrors && $this->onlyAttributes === []) {
```

Comment:
This code incorrect work if `showAllErrors` is `true` and `onlyAttributes` is non-empty(). We recevied first errors instead all errors for defined attributes.

Reply:
The code is not incorrect, when we use `showAllErrors`, as its name indicates, all errors must be shown, they come in a list without a key that cannot be filtered, that is why it is called showAllErrors, it doesn't make sense to show all errors with filtering, and I told you about that @samdark

Reply:
`firstErrorsOfAttributes(['a', 'b', 'c'])`

Reply:
> `showAllErrors` is `true` and `onlyAttributes` is non-empty().

In case `showAllErrors` is true and `onlyAttributes` is non-empty() we should received all errors for defined attrbites. In current code returned only first errors. it's not right.

Reply:
@vjik If you are right, I fixed it, the correction is correct, also change the name of the method to `firstErrorsOfAttributes(['a', 'b', 'c'])`, to avoid confusion, since if showAllErrors is true, they should be shown all the mistakes.

Reply:
In `showAllErrors` part "all" meat is not all attributes, it's all errors for attributes.
`onlyAttrbiutes` — filter attributes.
Names are good, need only small change behavior.

Reply:
> In case is true and is non-empty we should received all errors for defined attrbites. In current code returned only first errors. it's not right.`showAllErrors``onlyAttributes`
> 
> I think and should be two unrelated options.`showAllErrors``onlyAttributes`
> 
> `showAllErrors` — show all errors for each attribute or first errors only.
> 
> `onlyAttrbiutes` — show errors only for defined attribtes.

I think that `showAllErrors()` should show all errors regardless if it is a first error or not, so filtering should only be used when `showAllErrors()` is `false`.

Reply:
> > In case is true and is non-empty we should received all errors for defined attrbites. In current code returned only first errors. it's not right.`` showAllErrors``onlyAttributes ``
> > I think and should be two unrelated options.`` showAllErrors``onlyAttributes ``
> > `showAllErrors` — show all errors for each attribute or first errors only.
> > `onlyAttrbiutes` — show errors only for defined attribtes.
> 
> I think that `showAllErrors()` should show all errors regardless if it is a first error or not, so filtering should only be used when `showAllErrors()` is `false`.

This is don't allow get all errors for defined attributes only.

Reply:
> In `showAllErrors` part "all" mean is not all attributes, it's all errors for attributes. `onlyAttrbiutes` — filter attributes. Names are good, need only small change behavior.

done.

Reply:
Now `only Attributes()` will show errors only for the selected attributes, be they `showAllErrors()` or `firtsErrors()`.